server/src/SentryLogging.hs

```diff
@@ -0,0 +1,47 @@
+-- | Module for logging (crash) reports to Sentry
+module SentryLogging(
+  getCrashLogger, logException, logCrashMessage, runWithCrashLogger
+) where
+
+import Control.Exception
+import System.Environment (lookupEnv)
+
+import qualified System.Log.Raven as Sentry
+import qualified System.Log.Raven.Transport.HttpConduit as Sentry
+import qualified System.Log.Raven.Types as Sentry
+
+-- | Returns a Maybe SentryService which can be used to send error information
+-- to Sentry. Return value is Nothing is the environment variable SENTRY_DSN is
+-- not set.
+getCrashLogger :: IO (Maybe Sentry.SentryService)
+getCrashLogger = do
+    let
+    maybeSentryDSN <- lookupEnv "SENTRY_DSN"
```

Comment:
Getting the Sentry DSN should be part of the Config module, so that we have all configuration-related stuff together in one place. That would also allow specifying it as command line argument, if desired.
You can have a look at how the `JWT_SECRET` is handled in the config.